app/src/main/java/com/quran/labs/androidquran/ui/PagerActivity.java

```diff
@@ -1078,7 +1078,6 @@ private void updateActionBarTitle(int page) {
     String sura = QuranInfo.getSuraNameFromPage(this, page, true);
     ActionBar actionBar = getSupportActionBar();
     if (actionBar != null) {
-      translationsSpinner.setVisibility(View.GONE);
```

Comment:
@ozbek You are right, may be the problem is in [this](https://github.com/quran/quran_android/blob/master/app/src/main/java/com/quran/labs/androidquran/ui/PagerActivity.java#L771) line, I'll close this PR for now and try to use `updateActionBarSpinner()` instead of `updateActionBarTitle()`.